services/storage/tests/unit/test_dsm_dsmcleaner.py

```diff
@@ -43,7 +43,9 @@ def disabled_dsm_cleaner_task(monkeypatch: pytest.MonkeyPatch):
 
 @pytest.fixture
 def simcore_directory_id(simcore_file_id: SimcoreS3FileID) -> SimcoreS3FileID:
-    return SimcoreS3FileID(SimcoreS3DirectoryID.from_simcore_s3_object(simcore_file_id))
+    return SimcoreS3FileID(
+        Path(SimcoreS3DirectoryID.from_simcore_s3_object(simcore_file_id))
```

Comment:
is the linter happy with this?? I did not know that it would implicitly convert to a string..

Reply:
No issue, `SimcoreS3FileID`'s constructor expects an object.